src/Intercom/Clients/SegmentsClient.cs

```diff
@@ -59,19 +59,24 @@ public Segments List(Dictionary<String, String> parameters)
             return result.Result;
         }
 
-        public Segment View(String id)
+        public Segment View(String id, bool includeCount = false)
         {
             if (String.IsNullOrEmpty(id))
             {
                 throw new ArgumentNullException(nameof(id));
             }
 
+            Dictionary<String, String> parameters = new Dictionary<String, String>
+            {
+                { "include_count", includeCount.ToString() }
+            };
```

Comment:
I tested this locally and the problem with this approach is that we will always send the attribute even if it's false. So the params we get in the server when this is sent like this: `Segment segments = segmentsClient.View("58c67482ac0f91a20446037e");` are:

```
"params":{"include_count":"False","id":"58c67482ac0f91a20446037e"}
```

So this will return the count anyway. Can we instead of defining it as false, actually define as null? Maybe something like this:

```cs
public Segment View(String id, bool? includeCount = null)
{
    if (String.IsNullOrEmpty(id))
    {
        throw new ArgumentNullException(nameof(id));
    }

    Dictionary<String, String> parameters = new Dictionary<String, String>();
    if (includeCount != null && includeCount.HasValue)
            {
                parameters.Add("include_count", includeCount.ToString());
            }
```